Libraries/LibMarkdown/Text.h

```diff
@@ -46,14 +48,18 @@ class Text final {
         Style style;
     };
 
+    explicit Text(Text&& text) : m_spans(move(text.m_spans)) {}
```

Comment:
Is there any need for writing this constructor explicitly? It should get autogenerated.

Reply:
I'm not sure, if I remove it, the `Paragraph` constructor complains about this: ![](https://i.imgur.com/hr0MBcH.png)

Reply:
So apparently it doesn't generate the move constructor if you delete the copy one?

Does writing `= default` work? And why make it `explicit`?

Reply:
Having the constructor be `Text(Text&& text) = default;` makes it work. Should I leave it like that?

Reply:
Yeah